StripePaymentsUI/StripePaymentsUI/Resources/StripePaymentsUI.xcassets/BECS/anz.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "anz.pdf",
```

Comment:
1. in this folder, many of the images are marked as .pdf extension
2. Curious as to why we are using .imageset instead of .xcasset?

Reply:
It is an xcasset, the imageset is within the xcasset